ext/cl_exp_tensor.asciidoc

```diff
@@ -418,11 +421,131 @@ an abstract function that accesses a tensor element in its storage at
 given coordinate. The method how the coordinates translate to tensor
 storage addresses is unspecified.
 
+*clEnqueueTranslateFsomTensor* and *clEnqueueTranslateToTensor*
```

Comment:
Typo Fsom

Reply:
Maybe import/export are even better word here instead of 'translate'?

Reply:
Fixed type and renamed.